test/test_ops.py

```diff
@@ -1577,6 +1577,121 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+    @onlyCUDA
+    def test_opinfo_aten_ops_coverage(self):
```

Comment:
Why instantiate the test like this instead of per op? I'm not saying it's wrong, but what are the pros/cons?